.github/workflows/benchmark.yml

```diff
@@ -20,7 +20,7 @@ jobs:
           cache-dependency-path: 'npm-shrinkwrap.json'
           check-latest: true
       - name: Install dependencies
-        run: npm install --production --no-audit
+        run: npm ci --no-audit
```

Comment:
`husky` is now an indirect dependency (through `@netlify/eslint-config-node`) instead of being direct.

Binaries of indirect dependencies are symlinked by `npm` to the top-level `node_modules/.bin/` thanks to deduping. However, this seems to only work without the `--production` flag.

Also, this is more consistent with the other GitHub actions which all use `npm ci --no-audit`. Please note this is only for the GitHub action which computes the package size.

Reply:
Doesn't this give the wrong result now, though? It now includes devDependencies :/

Reply:
Oh right!

Reply:
@ehmicky How about doing `npm prune --prod` after `npm ci`? It's not ideal but it should be close to what it was before :)

Reply:
Done in https://github.com/netlify/cli/pull/4066